hwdb.d/60-master-of-seat.hwdb

```diff
@@ -0,0 +1,20 @@
+# This file is part of systemd.
+#
+# This file lists graphic devices that don't have a DRM driver and fall back to
+# a frame-buffer one instead. Since commit 6260d28b8a, frame-buffer devices are
+# no more considered as graphical capable.
+#
+# The matches have the 'fb' prefix to make sure that only the framebuffer
+# device, and not the (parent) PCI device, is tagged with 'master-of-seat'.
+#
+# Allowed properties are:
+#    ID_TAG_MASTER_OF_SEAT=1
+
+#########################################
+# eXtreme Graphic Innovation (XGI)
+#########################################
+
+# Z7/Z9 (XG20 core), a rather ancient graphic chip, doesn't have a DRM driver
+# and falls back to vesafb.
+fb:pci:v000018CAd00000020*
+ ID_TAG_MASTER_OF_SEAT=1
```

Comment:
as suggested please move the parallels and hyperv entry here too, and do them the same way

Reply:
According to commit  8e3480ebc097a8336633436f412a9c08902962ad, there's no kernel driver for Parallels video adapter hence no framebuffer driver, which is a prerequisite for using the hwdb, no ?

Reply:
i think it tries to say that because there's no explicit driver vesafb is being used, like for your case

Reply:
Ok in that case it makes sense to do the same for Parallels indeed.

Reply:
But note that the rule for Parallels tagged the PCI device, ie not the framebuffer.